docs/reference/runtime/runtime.md

```diff
@@ -1,2 +1,13 @@
 ## Runtime overview
 
+To start developing applications using Mbed OS, you need to create a main function and start coding. In most cases, no special setup is necessary.
+
+Write your Mbed OS applications in C++ (C++98).
+
+### Compilers
+
+Mbed OS supports:
+
+- GNU Arm Embedded Toolchain (GCC) version 6.
+- Arm Compiler version 5 and 6.
```

Comment:
I am not certain listing both here. As currently 5 is supported (ARM6 is there but fails for various targets and missing prebuilt libraries that needs updating). 

I would assume ARM5 (ARM6 experimental) currently, in 5.7 only ARM6.